### PR TITLE
Exporter/OcAgent: Add methods for getting and applying config.

### PR DESCRIPTION
**Pull request by songy23**
Split from #1476, to make that PR shorter.

### CODE REVIEW

File: exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java [hunk 1]
```diff
@@ -106,10 +105,9 @@ public AttributeValue apply(Double doubleValue) {
    *
    * @param spanData the {@code SpanData}.
    * @return proto representation of {@code Span}.
-   * @since 0.17
    */
   @SuppressWarnings("DefaultCharset")
```
Comment (bogdandrutu) [inline exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java]: We should probably have reproducible builds so we need to use a charset here if that is needed.

File: exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java [hunk 2]
```diff
@@ -378,5 +375,26 @@ public static TraceParams fromTraceConfigProto(
     return builder.build();
   }
 
+  // Creates a CurrentLibraryConfig message with the given Node and current TraceParams.
+  static CurrentLibraryConfig getCurrentLibraryConfig(
+      @javax.annotation.Nullable Node node, io.opencensus.trace.config.TraceConfig traceConfig) {
```
Comment (bogdandrutu) [inline exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java]: Probably better to return the TraceConfig proto here and avoid adding the only first time send the node logic in this util class.

File: exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java (hunk 1 above)
Comment (songy23) [reply exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java]: Removed - this seems to be I forgot to remove this annotation. It's no longer applicable after using `ByteString.copyFrom(bytes);`.

File: exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java (hunk 2 above)
Comment (songy23) [reply exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java]: Make sense, done.

File: exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java [hunk 3]
```diff
@@ -378,5 +371,21 @@ public static TraceParams fromTraceConfigProto(
...
 
+  // Creates a TraceConfig proto message with current TraceParams.
+  static TraceConfig getCurrentTraceConfig(io.opencensus.trace.config.TraceConfig traceConfig) {
+    TraceParams traceParams = traceConfig.getActiveTraceParams();
+    return toTraceConfigProto(traceParams);
+  }
+
+  // Creates an updated TraceParams with the given UpdatedLibraryConfig message and current
+  // TraceParams, then applies the updated TraceParams.
+  static void applyUpdatedConfig(
```
[the remark this answers is not in the document]
Comment (songy23) [reply exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java]: Make sense, done.

**Pull request merged**